test/Libraries/RevitIntegrationTests/FamilyTests.cs

```diff
@@ -37,6 +40,45 @@ public void GetFamilyInstancesByType()
         }
 
         [Test]
+        [TestModel(@".\Family\GetFamilyInstancesByType.rvt")]
+        public void GetFamilyInstancesByName()
+        {
+            var model = ViewModel.Model;
+
+            string samplePath = Path.Combine(workingDirectory, @".\Family\GetFamilyInstancesByName.dyn");
+            string testPath = Path.GetFullPath(samplePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+
+            RunCurrentModel();
+
+            //assert number of loaded families in document:
+            var fec = new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument);
+            fec.OfClass(typeof(Autodesk.Revit.DB.Family));
+
+            // obtain the family type with the provided name
+            var families = fec.Cast<Autodesk.Revit.DB.Family>();
+            var oldcount = families.Count();
+
+            //find the node
+            var node = this.Model.CurrentWorkspace.Nodes.Where(currentNode => currentNode.Name == "Family.ByName").FirstOrDefault();
+            Assert.IsNotNull(node);
+            //now delete the node
+            model.CurrentWorkspace.RemoveAndDisposeNode(node);
+
+            //now assert the families count is the same.
+            var fec2 = new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument);
+            fec2.OfClass(typeof(Autodesk.Revit.DB.Family));
+
+            // obtain the family type with the provided name
+            var families2 = fec2.Cast<Autodesk.Revit.DB.Family>();
+            var newcount = families.Count();
+
+            Assert.AreEqual(oldcount, newcount);
```

Comment:
The comment about asserting should probably be here for clarity.

Reply:
Agree